client/extensions/woocommerce/state/sites/locations/selectors.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import { find, flatMap, get, isArray, isEmpty, omit, sortBy } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { LOADING } from 'woocommerce/state/constants';
+
+/**
+ * @param {Object} state Whole Redux state tree
+ * @param {Number} [siteId] Site ID to check. If not provided, the Site ID selected in the UI will be used
+ * @return {Array} The locations tree, as retrieved from the server. It can also be the string "LOADING"
+ * if the locations are currently being fetched, or a "falsy" value if that haven't been fetched at all.
+ */
+export const getRawLocations = ( state, siteId = getSelectedSiteId( state ) ) => {
```

Comment:
Is this being used somewhere else?  From this PR it is not evident why it is being exported